app-tracking-protection/vpn-impl/src/main/res/values/strings-vpn.xml

```diff
@@ -207,7 +208,22 @@
     <string name="atp_ActivityTrackersCompanyBlockedOtherTimesOneCompany"><![CDATA[%1$d tracking attempts blocked from %2$d company in <b>%3$s</b> app]]></string>
     <string name="atp_ActivityTrackersCompanyBlockedOtherTimesOtherCompanies"><![CDATA[%1$d tracking attempts blocked from %2$d companies in <b>%3$s</b> app]]></string>
 
+    <!-- TODO [ANA] New strings added, need translation -->
```

Comment:
Please ignore the strings and TODOs. They are just a marker for me and will be cleaned up in the last PR.